packages/nextjs/styles/challenge-styles.css

```diff
@@ -0,0 +1,23 @@
+:root,
```

Comment:
I moved the new styles I had added into this new file. This is to prepare for extensions, and you cannot edit the `globals.css` file when making extensions.